docs/content/en/install/local/multi-cluster-helm/_index.md

```diff
@@ -0,0 +1,832 @@
+     --create-namespace --set installCRDs=true
+```
+
+**Output:**
+
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
+
+Then enter:
+
+```bash
+kubectx kind-k8ssandra-3
+
+helm install cert-manager jetstack/cert-manager --namespace cert-manager \
+     --create-namespace --set installCRDs=true
+```
+
+**Output:**
+
+```bash
+NAME: cert-manager
+LAST DEPLOYED: Thu Jan 27 15:28:59 2022
+NAMESPACE: cert-manager
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
+NOTES:
+cert-manager v1.7.0 has been deployed successfully!
+```
+
+### Install K8ssandra Operator in the control-plane
+
+First, you'll need to have [Helm v3+](https://helm.sh/docs/intro/install/) installed.
+
+Then configure the K8ssandra Helm repository:
+
+```console
+helm repo add k8ssandra https://helm.k8ssandra.io/stable
+```
+
+Update your Helm repository cache:
+
+```console
+helm repo update
+```
+
+Verify that you see the `k8ssandra-operator` chart:
+
+```console
+helm search repo k8ssandra-operator
+```
+
+In the following example, of the four clusters we've created in the section above, we'll use `kind-k8ssandra-0` as our control-plane.
+
+```bash
+kubectx kind-k8ssandra-0
+
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+     --create-namespace
+```
+
+### Install K8ssandra Operator in the data-planes
+
+In this example, we'll use the three other clusters as data-planes.
+
+```bash
+kubectx kind-k8ssandra-1
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+ --create-namespace
+
+kubectx kind-k8ssandra-2
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+ --create-namespace
+
+kubectx kind-k8ssandra-3
+helm install k8ssandra-operator k8ssandra/k8ssandra-operator -n k8ssandra-operator \
+ --create-namespace
+```
+
+### Verify control-plane configuration
+
+```
+kubectx kind-k8ssandra-0
+
+kubectl -n k8ssandra-operator get deployment k8ssandra-operator \
+ -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
+```
+
+**Output:**
+
+```bash
+true
+```
+
+### Verify data-plane configuration
+
+We could test for `K8SSANDRA_CONTROL_PLANE`, which for each of the three clusters in our example serving as data-planes, should return `false`. Just one example:
+
+```
+kubectx kind-k8ssandra-1
+
+kubectl -n k8ssandra-operator get deployment k8ssandra-operator \
+ -o jsonpath='{.spec.template.spec.containers[0].env[?(@.name=="K8SSANDRA_CONTROL_PLANE")].value}'
+```
+
+**Output:**
+
+```bash
+false
+```
+
+### Generate and install ClientConfigs
+
+[create-clientconfig.sh](https://github.com/k8ssandra/k8ssandra-operator/blob/main/scripts/create-clientconfig.sh) lives in the k8ssandra-operator repo. It is used to configure access to remote clusters. 
```

Comment:
Let's add a note somewhere in here that mentions that the operator automatically restarts itself whenever there is a change to a ClientConfig (create/update/delete). This is done in order to update connections to remote clusters.

Reply:
Thanks - coming in my next commit.

Reply:
Done